### ISSUE TITLE
Some implementation details

### ISSUE THREAD

**Issue by zhangluustb** [gazeai/STDN-PyTorch#5]
1.
https://github.com/gazeai/STDN-PyTorch/blob/4fbe6dd713d87900559c84a8d01b6024bf567ced/model.py#L477

synth_cat = torch.cat([synthesized_images[0], reconstructed_images[0][len(x) // 2:]], dim=0).float()
Is the order of "live and spoof" correct？

2.

Why set a new network called no_grad_gen?
